desktop_version/src/VFormat.c

```diff
@@ -0,0 +1,463 @@
+#include "VFormat.h"
+
+#include <physfs.h>
+#include <SDL.h>
+#include <stdbool.h>
+
+#include "CWrappers.h"
+
+
+static inline bool is_whitespace(char ch)
+{
+    return ch == ' ' || ch == '\t';
+}
+
+static inline void strip_whitespace(const char** string, size_t* bytes)
```

Comment:
Conceptually, this function modifies a string (as defined by a pointer and the length) to strip leading and trailing whitespace. It's just that this modification is done by advancing the pointer and decreasing the length, instead of actually changing/moving any characters themselves. So where this function is used, we had `arg_name`+`arg_name_len` and `arg_type`+`arg_type_len` with possible leading and/or trailing whitespace, and after the function calls, these strings no longer have their whitespace.

Reply:
Ah okay, that makes a lot more sense.

Then maybe it should be named something like `strip_leading_and_trailing_whitespace`. Or if that's too long, then at least put a comment in the function saying something like `/* Strips leading and trailing whitespace. */`.

Reply:
Hm, so I thought "strip" would already be a big hint since it's a common term for removing leading and trailing whitespace (with `_whitespace` making it more clear), but it looks like "trim" may be a bit more common. PHP has `trim(string)`, Java has `String.trim()` and `String.strip()` (with slightly different behavior as to what is considered whitespace, apparently), JavaScript has `string.trim()`, C# has `String.Trim()`. From what I can quickly find thus far only Python has `str.strip()` exclusively.

I'll rename the function to `trim_whitespace` because that seems a little bit clearer from the start, and the comment definitely couldn't hurt.